packages/client/src/features/accessibility/README.md

```diff
@@ -0,0 +1,64 @@
+# Keyboard-Only: Accessibility Features
+
+The features presented in the following are introduced for disability-aware conceptual modeling and allow basic user model interactions to be performed only with the keyboard.
+
+## Getting started
+
+In order to activate and use all of the currently provided keyboard only accessibility features, the module `glspAccessibilityModule` needs to be in use.
+To activate and use only specific accessibility features, the respective module (see list below) needs to be in use.
+
+## Features
+
+Below is a list of the keyboard only accessibility features:<br>
+
+| Feature |                                           Description                                            |             Shortcut              |          Module           |
+| ------- | :----------------------------------------------------------------------------------------------: | :-------------------------------: | :-----------------------: |
+| Search  | Search for elements or edges using labels, types, etc. <br> with respective diagram highlighting |             `CTRL+F`              | `glspSearchPaletteModule` |
```

Comment:
Just a suggestion: You could use for the shortcuts in this document also the keybinding tag, e.g. `<kbd>CTRL</kbd>+<kbd>F</kbd>` to get <kbd>CTRL</kbd>+<kbd>F</kbd>